src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -310,7 +310,7 @@ export const NavBar: React.FC<NavBarPropsType> = ({
           <div className={navBarScheduleClassName}>
             <NavBarSchedule
               isVisible={isEventScheduleVisible}
-              venueId={venueId}
+              venue={currentVenue}
```

Comment:
the functionality of the NavBarSchedule changed a bit, and we need the name of the current venue now.
So I find it ok to get the venue object from the parent component, especially taking into account that the NavBar has it anyway.

Reply:
> , especially taking into account that the NavBar has it anyway.

☝️ 
Adding tighter coupling between two components for the price of not repeating a hook?

Reply:
Example:

A has id, A contains child C, A passes id to child.
A has hook anyway, A decides to pass object instead of id to C, C props get changed, A passes object to C.
*** some time later ***
B has id, B needs to add child C, B can't pass id to child, B needs to add hook for the object, B passes object to C